packages/picasso/src/DatePicker/styles.ts

```diff
@@ -1,8 +1,15 @@
-import { createStyles } from '@material-ui/core/styles'
+import { createStyles, Theme } from '@material-ui/core/styles'
 
-export default () =>
+export default ({ palette, shadows }: Theme) =>
   createStyles({
     calendar: {
       outline: 'none',
     },
+    footer: {
+      backgroundColor: palette.grey.lightest,
+      boxShadow: shadows[5],
+      borderRadius: '0 0 4px 4px',
```

Comment:
please import `sizes` from the `Theme` and use it here as well
```suggestion
      borderRadius: `0 0 {sizes.borderRadius.small} {sizes.borderRadius.small}`,
```

Reply:
Done!